pontoon/administration/static/js/admin_project.js

```diff
@@ -83,7 +83,7 @@ $(function() {
       return;
     }
 
-    button.addClass('in-progress').html('Pretranslating...');
+    button.addClass('in-progress').html('Pretranslating');
```

Comment:
Now I see that we're doing the exact same thing with the Sync button. :/

So I suggest you revert this change (i.e. restore ...) and increase the width of buttons from 130px to 150px.

Reply:
I should have pointed it out earlier. What about the actual async behavior? Is it working as expected?